config/projects.yml

```diff
@@ -191,8 +191,8 @@ doctrine-phpcr-admin-bundle:
       php: ['5.3', '5.4', '5.5', '5.6', '7.0', '7.1']
       versions:
         symfony: ['2.3', '2.7', '2.8']
-        sonata_admin: ['2']
```

Comment:
Ooooh I forgot I changed it instead of doing just a removal. No idea why the admin job builds though.